src/twitch.js

```diff
@@ -32,30 +34,53 @@ Twitch.prototype.makeRequest = function(http) {
 }
 
 /**
-* @description : gets user data from the api
-* @param {String} username : the username we want information from
-* @returns {Promise.<string, Error>} : resolves JSON data or rejects an error
-*/
-Twitch.prototype.getUser = function(username) {
+ * @method getUserId
```

Comment:
I know nothing about nock tests. Should this just be added into the current test?